flow/connectors/core.go

```diff
@@ -224,6 +224,9 @@ type RenameTablesConnector interface {
 }
 
 func GetConnector(ctx context.Context, config *protos.Peer) (Connector, error) {
+	if config == nil {
```

Comment:
why are we passing nil config?

Reply:
not required

Reply:
should remove, bugs should be panics not errors